src/transformers/pipelines/zero_shot_audio_classification.py

```diff
@@ -43,8 +43,8 @@ class ZeroShotAudioClassificationPipeline(Pipeline):
     >>> dataset = load_dataset("ashraq/esc50")
     >>> audio = next(iter(dataset["train"]["audio"]))["array"]
     >>> classifier = pipeline(task="zero-shot-audio-classification", model="laion/clap-htsat-unfused")
-    >>> classifier(audio, candidate_labels=["Sound of a dog", "Sound of vaccum cleaner"])
-    [{'score': 0.9995999932289124, 'label': 'Sound of a dog'}, {'score': 0.00040007088682614267, 'label': 'Sound of vaccum cleaner'}]
+    >>> output = classifier(audio, candidate_labels=["Sound of a dog", "Sound of vaccum cleaner"])
+    >>> # [{'score': 0.9995999932289124, 'label': 'Sound of a dog'}, {'score': 0.00040007088682614267, 'label': 'Sound of vaccum cleaner'}]
```

Comment:
I think it's possible to output the values. Any reason to not output?

Reply:
Because it seeks like depending on torch / hardware version there is a 0.00000001 difference in rounding outputs 🫠 that’s why it is still failing.
We could use nested input but feels confusing for random users no?

Reply:
Oh, you can just use 4 digits, like `0.9996` and `0.0004`. It should work well.

Reply:
Oh okay! I thought `nested_simplify` was required for that! Thanks for the tip